recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ outputs:
         - python {{ python_min }}
         - pip
         - setuptools
-        - setuptools_scm !=6.1.0
+        - setuptools_scm !=6.1.0,<=8.3.0  # upper constrain to avoid circular dependencies
```

Comment:
Will this pin to `<=8.3.0` stay like this forever? This seems like it could mean trouble down the road...

Reply:
as it is just for build it should be fine(ish). But the other option is to patch ``importlib-metadata`` , remove the dependency of setuptools-scm and add the version file manually.